### PR TITLE
Updated README.md with Resources

### PR DESCRIPTION
**Pull request by OO7kartik**
Added the resources.

### CODE REVIEW

**Comment by OO7kartik** [comment]
On it :)

File: AlgoRhythm/README.md [hunk 1]
```diff
@@ -32,7 +32,74 @@ of the algorithmic principles and how to solve a given problem in the most optim
...
+
+2. Questions and Resources : 
+* [CodeMonk](https://www.hackerearth.com/practice/codemonk/)
+
+3. Game Theory :
+* Nim
+* Sprague-Grundy Theorem
+* First try this [**question**](https://www.hackerrank.com/challenges/game-of-stones-1/problem) without reading any theory.
+
+4. Theory :
```
Comment (AndyS12) [inline AlgoRhythm/README.md]: Do not number this line. It's sub-topic

Comment (OO7kartik) [reply AlgoRhythm/README.md]: got it theory and questions are sub-topic

**Comment by gargarchit** [comment]
@OO7kartik  I Added Some changes like I add a separate file for Resources and PR on your Forked Repo 
Do check it.

**Comment by shoebpate1** [comment]
hmm, @OO7kartik merge his work. will merge this one after that.

**Comment by OO7kartik** [comment]
i've merged everything, i liked the idea :)

**Comment by gargarchit** [comment]
Yeah

**Pull request merged**